gef-ui/server/utils/helpers.test.js

```diff
@@ -25,21 +25,17 @@ import { getFacilityCoverStartDate } from './facility-helpers';
 
 import { MOCK_ISSUED_FACILITY, MOCK_FACILITY, MOCK_ISSUED_FACILITY_UNCHANGED, MOCK_UNISSUED_FACILITY } from './mocks/mock-facilities';
 
-import { makerCanReSubmit } from './deal-helpers';
```

Comment:
Why has this been removed please?

Reply:
This shouldn't have been tested in this file, I moved these tests to `deal-helpers.test.js`

Reply:
Ah sorry didn't spot!